.github/workflows/ccpp.yml

```diff
@@ -54,7 +54,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        llvm: ["7", "8", "9", "10", "11", "12", "13", "14", "15"]
+        llvm: ["7", "8", "9", "10", "11", "12", "13"]
```

Comment:
why are u disabling tests?

Reply:
It was mistakenly added last commit (automerged without realizing). These arent easily installable on ubuntu 18.04, so worth removing (since they are tested on 20.04)

Reply:
I missed the fact that those changes only apply to 18